lib/src/objects/parse_base.dart

```diff
@@ -3,7 +3,7 @@ part of flutter_parse_sdk;
 abstract class ParseBase {
   String parseClassName;
   Type type;
-  bool _dirty = false; // reserved property
+  final bool _dirty = false; // reserved property
```

Comment:
This feels weird. Why is this final. 

This might not be related to what you are doing here, but does seem weird that we even set, _dirty and finalise it.

Reply:
currently in your code you didnt modify it. so I just follow dart's lint to modify it to final.